controllers/feeds/getFeedChat.js

```diff
@@ -120,7 +122,7 @@ const finalize = (req, id, myReaction, newGroup, activity_id, constantActor, kar
   if (myReaction) {
     myReaction = {
       ...myReaction,
-      isOwningReaction: req.userId === myReaction.user_id || myReaction.user_id === id
+      isOwningReaction: req.userId === myReaction.user_id
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for `isOwningReaction` has been simplified to only check if `req.userId` equals `myReaction.user_id`. This change seems to be in line with the PR's purpose of separating anonymous and signed comment reactions.
```diff
- isOwningReaction: req.userId === myReaction.user_id || myReaction.user_id === id
+ isOwningReaction: req.userId === myReaction.user_id
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->